fairlearn/datasets/_fetch_adult.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import pathlib
+
+from sklearn.datasets import fetch_openml
+from ._common import _DOWNLOAD_DIRECTORY_NAME
+
+
+def fetch_adult(*, cache=True, data_home=None,
+                as_frame=False, return_X_y=False):
+    """Load the UCI Adult dataset (binary classification).
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total             48842
+    Dimensionality               14
+    Features                   real
```

Comment:
Is that true? I don't think that the features are real valued: https://archive.ics.uci.edu/ml/datasets/Adult

Reply:
The feature `fnlwgt` is real valued. Everything else isn't, although a few others are continuous.